cypress/integration/pages/frontPage/tests.js

```diff
@@ -117,8 +115,37 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
           });
         });
 
-        if (serviceHasIndexAlsos(service)) {
-          it('should contain Index Alsos at a mobile view', () => {
+        it('should contain Index Alsos at a mobile and desktop view, if relatedItems block exists', () => {
+          const serviceHasRelatedItems = () => {
+            cy.request(`${config[service].pageTypes.frontPage.path}.json`).then(
+              ({ body }) => {
+                const topstories = body.content.groups[0];
+                const relatedItemsExists = Object.prototype.hasOwnProperty.call(
```

Comment:
I think this could maybe be replaced with this maybe?
```
const relatedItemsExists = 'relatedItems' in topstories
```

Reply:
@dr3 I started off adding your suggestion this morning and then I ended up completely changing how this test runs and what it looks for 🙈

another review would be great 😄